domain-models-runtime/src/main/java/org/folio/rest/persist/PostgresClient.java

```diff
@@ -3517,7 +3518,13 @@ public void dropCreateDatabase(String database) throws SQLException {
    */
   public Future<List<String>> runSQLFile(String sqlFile, boolean stopOnError) {
     Promise<List<String>> promise = Promise.promise();
-    runSQLFile(sqlFile, stopOnError, promise.future());
+    runSQLFile(sqlFile, stopOnError, (res) -> {
+        if (res.failed()) {
+          log.error("Error while executing SQL :" + res.cause().getMessage());
+          promise.fail(res.cause());
+        }
+        promise.complete(res.result());
+        });
```

Comment:
This change is not needed because runSQLFile(String, boolean, Handler) already has logged the error.

Reply:
done.